pkg/pods/pods.go

```diff
@@ -133,7 +144,81 @@ func (obj *Pods) Create(options PodOptions) (k8sTypes.Pod, error) {
 	if err != nil {
 		return k8sTypes.Pod{}, err
 	}
-	return *pod, nil
+
+	if options.Wait == "" {
+		return *pod, nil
+	}
+	waitOpts := WaitOptions{
+		Name:      options.Name,
+		Namespace: options.Namespace,
+		Status:    Running,
+		Timeout:   options.Wait,
+	}
+	status, err := obj.Wait(waitOpts)
+	if err != nil {
+		return k8sTypes.Pod{}, err
+	}
+	if !status {
+		return k8sTypes.Pod{}, errors.New("timeout exceeded waiting for pod to be running")
+	}
+
+	return obj.Get(options.Name, options.Namespace)
+}
+
+// Options for waiting for a Pod status
+type WaitOptions struct {
+	Name      string // Pod name
+	Namespace string // Namespace where the pod is running
+	Status    string // Wait until pod reaches the specified status. Must be one of "Running" or "Succeeded".
+	Timeout   string // Timeout for waiting condition to be true
+}
+
+// Wait for the Pod to be in a given status up to given timeout and returns a boolean indicating if the staus was reached. If the pod is Failed returns error.
+func (obj *Pods) Wait(options WaitOptions) (bool, error) {
+	if options.Status != Running && options.Status != Succeeded {
+		return false, errors.New("Wait condition must be 'Running' or 'Succeeded'")
```

Comment:
For linting rules (currently disabled, see issue #41 ), I _think_ `Wait ...` should be `wait ...` since it is not a resource type.